src/Api/Auth/Controllers/WebAuthnController.cs

```diff
@@ -118,7 +124,9 @@ public async Task Post([FromBody] WebAuthnLoginCredentialCreateRequestModel mode
 
     private async Task ValidateRequireSsoPolicyDisabledOrNotApplicable(Guid userId)
     {
-        var requireSsoLogin = await _policyService.AnyPoliciesApplicableToUserAsync(userId, PolicyType.RequireSso);
+        var requireSsoLogin = _featureService.IsEnabled(FeatureFlagKeys.PolicyRequirements)
+            ? (await _policyRequirementQuery.GetAsync<RequireSsoPolicyRequirement>(userId)).CanUsePasskeyLogin
+            : await _policyService.AnyPoliciesApplicableToUserAsync(userId, PolicyType.RequireSso);
```

Comment:
I think the boolean is flipped here because of the change in interfaces.

Today the error is thrown if `requireSsoLogin == true`, but after the refactor it would be thrown if `CanUsePassKeyLogin == false`.

It might be clearer to make a `vNext` method.